torch/nn/init.py

```diff
@@ -241,10 +276,11 @@ def xavier_normal_(tensor, gain=1):
         >>> w = torch.empty(3, 5)
         >>> nn.init.xavier_normal_(w)
     """
+    # type: (Tensor, float) -> Tensor
     fan_in, fan_out = _calculate_fan_in_and_fan_out(tensor)
-    std = gain * math.sqrt(2.0 / (fan_in + fan_out))
-    with torch.no_grad():
-        return tensor.normal_(0, std)
+    std = gain * math.sqrt(2.0 / float(fan_in + fan_out))
```

Comment:
Why did you need the float cast ?

Reply:
`fan_in` and `fan_out` are ints, and there's no `aten::div(float, int)` currently

Reply:
Can you add that to this PR? it should be like a two line change - `DEFINE_INT_FLOAT_OP(aten::div, a / b, float)` in register_prim_ops and add '/' to test_number_math.

Reply:
This wasn't really resolved.

Reply:
At least should be left for a follow up, I feel like it counts as an implicit `int` -> `float` conversion which if we're going to add should be at the compiler level instead of per-op.